test/sdk-benchmarks/pom.xml

```diff
@@ -48,8 +48,8 @@
         <uberjar.name>benchmarks</uberjar.name>
 
         <sdk-v1.version>1.11.404</sdk-v1.version>
-        <jackson.version>2.12.3</jackson.version>
-        <jackson.databind.version>2.12.3</jackson.databind.version>
+<!--        <jackson.version>2.12.3</jackson.version>-->
+<!--        <jackson.databind.version>2.12.3</jackson.databind.version>-->
```

Comment:
Can we remove the commented-out lines?

Reply:
Yes! I meant to do it but apparently forgot...